css/main.css

```diff
@@ -1,3 +1,373 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+header {
+  padding-top: 50px;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 150px;
+  background-color: white;
+}
+
+.draw-title {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+}
+
+.main-article {
+  margin-left: 10px;
+  font-size: 2rem;
+  font-weight: 700;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.sec-article {
+  margin-left: 5px;
+  font-size: 2rem;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.the-nav {
+  margin-right: 10%;
+  margin-left: 22%;
+}
+
+.blue-square {
+  background-color: blue;
+  min-width: 20px;
+  height: 20px;
+  margin-left: 5%;
+  margin-top: 15px;
+}
+
+.nav-list {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  padding: 2%;
+  justify-content: space-between;
+  list-style-type: none;
+  font-size: 1.2rem;
+}
+
+.burger-menu {
+  display: none;
+}
+
+a:link {
+  text-decoration: none;
+  color: black;
+}
+
+a:visited {
+  color: lightblue;
+}
+
+a:hover {
+  color: blue;
+  cursor: pointer;
+}
+
+a:active {
+  text-decoration: underline;
+  color: darkblue;
+}
+
+main {
+  margin-top: 200px;
+}
+
+.first-row {
+  height: 150px;
+}
+
+.gray-div {
+  background: rgb(216, 212, 190);
+}
+
+.light-gray-div {
+  background: rgb(240, 237, 224);
+}
+
+.photo-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.my-photo {
+  width: 300px;
```

Comment:
Can you adapt the size of your photo for smaller screens? It's currently too big between 570px and 1000px screen width